bundle/pythonic_usage_guidance/pythonic_bundle_access.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>would be great to add some URLs for the related docs.monai.io pages</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1501/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks, added it in the latest commit.